flang/include/flang/Parser/dump-parse-tree.h

```diff
@@ -581,6 +581,14 @@ class ParseTreeDumper {
   NODE(OmpAllocateClause::AllocateModifier, Align)
   NODE(parser, OmpScheduleClause)
   NODE_ENUM(OmpScheduleClause, ScheduleType)
+  NODE(parser, OmpInitClause)
+  NODE(parser, InteropType)
+  NODE_ENUM(InteropType, Kind)
+  NODE(parser, InteropPreference)
+  NODE(OmpInitClause, InteropPreferenceList)
```

Comment:
Better to replace InteropPreferenceList with InteropModifier to be in sync with the standard.

Reply:
Addressed the comment on [#1c57d1e](https://github.com/swatheesh-mcw/llvm-project/pull/1/commits/1c57d1ea2b8e7095a834689ad5b832e6a2a84c37)